### ISSUE TITLE
For accumulating bundles, validate latest version of collections only

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/validate#231]
When `pds4.bundle` rule is enabled, and the bundle references collections by LID, validate checks all versions of the associated collections.

Validate should have a new flag that enables the ability to check for latest collections only:

```
--latest-collections-only    Flag for enabling validate to check the latest version of a
                       collection (and associated primary products) to validate. By default, validate
                       checks all versions of the collection associated with the bundle. Must be used 
                       with the pds4.bundle and pds4.collection rules.
```

**Comment by jordanpadams** [comment]
This is actually a bug per the SR where if a bundle references a collection by LID, it should only perform integrity checking on the latest version of the collection. #256 is the new, more better ticket to track this.

**Comment by jordanpadams** [comment]
Closed per #238
